Classes/Editing/FLEXIvarEditorViewController.m

```diff
@@ -36,10 +36,13 @@ - (void)viewDidLoad
     [super viewDidLoad];
     
     self.fieldEditorView.fieldDescription = [FLEXRuntimeUtility prettyNameForIvar:self.ivar];
-    
-    FLEXArgumentInputView *inputView = [FLEXArgumentInputViewFactory argumentInputViewForTypeEncoding:ivar_getTypeEncoding(self.ivar)];
+
+    const char *typeEncoding = ivar_getTypeEncoding(self.ivar);
+    id currentValue = [FLEXRuntimeUtility valueForIvar:self.ivar onObject:self.target];
+    currentValue = [FLEXRuntimeUtility potentiallyUnwrapBoxedPointer:currentValue type:typeEncoding];
```

Comment:
I'm not sure we should try to unwrap NSValue instances here. It makes sense for object explorers because it's a read-only operation and you usually don't care about the "box" when you're trying to inspect something, but in this case if you happen to edit an ivar holding an NSValue which boxes a pointer to another object, the editor will display the unboxed object, and hitting "set" will change the ivar to the unboxed object even if the user didn't make any changes.

Thoughts?

Reply:
Good call on the setter issue.
So the 2 problem here:
1: without unwrapping, you basically cannot view the object/value at all since no proper input view could be found.
2: as you said, the setter is probably broken. Maybe we need some extra step there.

Reply:
I think we should special-case `FLEXArgumentInputStringView` to accept an `NSValue *` boxing a `char *`, unless someone wants to do some heavy-lifting and make `FLEXArgumentInputView` support unboxing and re-boxing `NSValue` instances, which might be a neat feature down the line but imo it's a very narrow use case outside of `char *`

Reply:
I will think about this again and give it another try.